runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -149,6 +149,11 @@ private void openGroupConfigPanel(ConfigDescriptor cd, ConfigManager configManag
 
 		for (ConfigItemDescriptor cid : cd.getItems())
 		{
+			if (cid.getItem().name().equals("hidden"))
```

Comment:
This should probably instead be a setting on ConfigItem. And in a separate commit from the rest of the username stuff.